### PR TITLE
[AKS] Withdraw hearing test

### PR DESCRIPTION
**Pull request by arompa**

### CODE REVIEW

File: e2e/utils/api.ts [hunk 1]
```diff
@@ -22,13 +24,50 @@ export class API {
...
+            body: JSON.stringify(body),
+            headers: API.headers,
+            resolveWithFullResponse: true
+        };
+        const response = await rp(options);
 
         return response.statusCode;
     }
 
+    static async deleteListingRequest(id: string) {
```
Comment (andrewtaylorkainos) [inline e2e/utils/api.ts]: private?

File: e2e/protractor.conf.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 // Protractor configuration file, see link for more information
 // https://github.com/angular/protractor/blob/master/lib/config.ts
 
-const { SpecReporter } = require('jasmine-spec-reporter');
+const {SpecReporter} = require('jasmine-spec-reporter');
 const puppeteer = require('puppeteer');
 const URL = require('./e2e-url.js');
 
-const isHeadlessModeEnabled = !!process.env.TEST_URL;
+const isHeadlessModeEnabled = !!process.env.TEST_URL;;
```
Comment (andrewtaylorkainos) [inline e2e/protractor.conf.js]: Extra semicolon :o

File: e2e/utils/api.ts (hunk 1 above)
Comment (arompa) [reply e2e/utils/api.ts]: this is used in the test afterAll to cleanup the environment

File: e2e/pages/view-hearing.po.ts [hunk 3]
```diff
@@ -17,4 +19,20 @@ export class ViewHearingPage {
     getHeaderText() {
         return this.header.getText();
     }
+
+    getStatusText() {
+        return this.status.getText();
+    }
+
+    async chooseAnActionFromDropDown(optionName: string) {
```
Comment (zielinskikamil) [inline e2e/pages/view-hearing.po.ts]: we have sth like this already in repo ^^ 
Take a look at *selectValueFromSingleSelectOption* which is in *ElementHelper*

File: e2e/utils/api.ts [hunk 4]
```diff
@@ -22,13 +24,50 @@ export class API {
...
+        await API.login();
+        const versionId =  (await API.getListingRequestById(id)).version;
```
Comment (zielinskikamil) [inline e2e/utils/api.ts]: *versionId* is a bit misleading. how about just *hearingVersion*?

**Pull request merged**